lessons/dealing-with-big-data-and-network-analysis-using-neo4j.md

```diff
@@ -0,0 +1,453 @@
+We are going to:
+
+- Place our CSV files so they can be accessed by Neo4j
+- Start the database.
+- Open the browser and connect to the database so we can run a **Cypher** command.
+- Using Cypher, we will load our specially formatted CSV files into the database.
+
+This process assumes that your data is cleanly separated into node and edge CSV files.
+
+## Moving the CSV files to the import directory
+
+Click on the "Manage" button in the database pane, then the drop down menu next to "Open Folders" and select "Import." A window will appear with a directory. 
+
+{% include figure.html filename="new-neo4j-files.png" caption="Pressing the Open Folders button" %}
+
+You now need to copy the 
+**nodes_nodes_companies.csv** and the **edges_director_duration.csv** files there.
+Now we can use a Cypher command to load the files.
+
+{% include figure.html filename="importing.png" caption="Moving the files to this directory" %}
+
+## Start the database
+
+In order to start the database, press the triangular play icon.
+
+{% include figure.html filename="new-neo4j-start-db.png" caption="Pressing the start database button" %}
+
+In the "Details" tab beneath, you will see information about the database starting.
+You'll notice that the database server is running on "HTTP port 7474".
+Neo4j allows access to the database through a web server. In the next step, we will open 
+a browser to connect to the database.
+
+## Opening the browser
+
+Click on the "Open Browser" button. A new browser window will appear.
+
+First, you will need to login to your new database. By default,
+ the username and password are both `neo4j`. After you log in the
+  first time, you will be prompted to create a new password.
+
+At the top of the window is a prompt with a blinking cursor. 
+We can add our Cypher command to load our data here
+
+{% include figure.html filename="new-neo4j-browser.png" caption="Browser window" %}
+
+### Using Cypher to load the CSV files into the database
+
+Again, press the trianglar run icon ">" in order to execute the command. An example can be seen in the screenshot below:
+
+{% include figure.html filename="new-neo4j-load1.png" caption="Cypher command window: loading nodes into the database" %}
+
+```sql
+LOAD CSV with headers FROM "file:///nodes_companies.csv" as nodes
+CREATE (n:COMPANY { id:toInteger(nodes.companyId), name:nodes.name })
+```
+
+The Cypher command LOADs the CSV file that contains informative headers (i.e. the first line)
+from the file we placed in the import directory. (By default, Neo4j can only load files from this directory.)
+The results will be stored as an object called **nodes**.
+
+The second line CREATEs data in our database. In particular, we create a series of node objects of the type COMPANY 
+that contain a **name** and an **id**. We set the name of this new company node to the name stored in the **nodes.name** object and the id to the same as stored in the **nodes.companyID**.
```

Comment:
For terms that represent programming language objects or commands, I'd generally wrap them in backticks, to distinguish them from the boldface used for more general vocabulary terms like **nodes**